docs/en/getting-started/step3.md

```diff
@@ -5,6 +5,21 @@ title: Importing Hosts into Qv2ray
 # Step 3: Importing Hosts into Qv2ray
 Qv2ray supports many ways of importing your configuration. To get you started, we have just picked these most common use cases.
 
+## Manual Configuration
```

Comment:
You may want to reorder the methods by most commonly used:
 - vmess (share link)
 - manually edit (in GUI)
 - import / edit (JSON)

Reply:
I agree with this.